operations/overlay_compute_ngd.yml

```diff
@@ -0,0 +1,36 @@
+id: overlay_compute_ngd
+subclass_of: overlay
+name: overlay compute ngd
+description: This operation computes the normalized Google distance (co-occurrence frequency) in PubMed abstracts and adds virual edges between qnodes AND/OR knodes AND/OR results edge bindings.
+examples: 
+- input: overlay/messages/07_input_ngd.json
+  output: overlay/messages/08_output_ngd.json
+input_requirements:
+- Input qgraph contains the two qnodes specified in 'subject_qnode_key' and 'object_qnode_key'
```

Comment:
This is now out of date. Also, I'm not sure how we ought to handle these sort of joint requirements between the TRAPI message and the operation parameters. My feeling is that this should not be described as an input requirement, but should just be documented in the parameters schema. In other words, the operation parameters can depend on the message, but not vice versa.

Reply:
TODO: remove this input requirement and double check the parameter documentation mentions the the qnode_keys parameter should be valid qnodes

Reply:
This has been done and the documentation/schema has been regenerated. If you think this is resolved @patrickkwang I think this can be merged.